Prace_domowe/Praca_domowa3/Grupa4/Marcin Wilk/PD3.ipynb

```diff
@@ -0,0 +1,1085 @@
+{
```

Comment:
<p>Fajnie, że sprawdziłeś specifity </p><p> </p><p>Jakbyś to ładniej pokazał, to byłoby gicik</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Wrrróć, pokiełbasiłeś te miary, to co nazywa się specifity, to jest recall</p>